midpoint/quickstart/index.adoc

```diff
@@ -13,9 +13,8 @@ To make the deployment as easy as possible for you, we provide a shell script yo
 
 To manage the environment for midPoint, the script uses link:https://docs.docker.com/compose/[Docker Compose] and multiple containers.
 
-The script is built for *Linux*, and it also works on macOS.
-If you're running another operating system, you may spend more time adjusting your system to satisfy the requirements.
-Another option is to install Linux on a virtual machine (e.g., using link:https://www.virtualbox.org/[VirtualBox]) and run the script on the virtual machine.
+The script is built for *Linux* and it also works on macOS.
+If you're running another operating system, you may spend more time adjusting your system to make the script useful to you.
```

Comment:
This is intended for beginners. We can't expect them to adjust the script. For Windows we can suggest docker desktop, that is yet to be documented.